apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/modules/components/Claimable.tsx

```diff
@@ -0,0 +1,646 @@
+      isErc721={isErc721}
+      isSequentialTokenIdInstalled={isSequentialTokenIdInstalled}
+      chainId={props.contract.chain.id}
+    />
+  );
+}
+
+export function ClaimableModuleUI(
+  props: Omit<ModuleCardUIProps, "children" | "updateButton"> & {
+    primarySaleRecipient: string | undefined;
+    isPending: boolean;
+    isOwnerAccount: boolean;
+    claimCondition: ClaimCondition | undefined;
+    update: (values: ConfigFormValues) => Promise<void>;
+    mint: (values: MintFormValues) => Promise<void>;
+    isErc721: boolean;
+    isSequentialTokenIdInstalled: boolean;
+    chainId: number;
+  },
+) {
+  return (
+    <ModuleCardUI {...props}>
+      <div className="h-1" />
+      {props.isPending && <Skeleton className="h-[74px]" />}
+
+      {!props.isPending && (
+        <div className="flex flex-col gap-4">
+          {/* Mint NFT */}
+          <Accordion type="single" collapsible className="-mx-1">
+            <AccordionItem value="metadata" className="border-none">
+              <AccordionTrigger className="border-border border-t px-1">
+                Mint NFT
+              </AccordionTrigger>
+              <AccordionContent className="px-1">
+                {props.isOwnerAccount && (
+                  <MintNFTSection
+                    mint={props.mint}
+                    isErc721={props.isErc721}
+                    isSequentialTokenIdInstalled={
+                      props.isSequentialTokenIdInstalled
+                    }
+                  />
+                )}
+                {!props.isOwnerAccount && (
+                  <Alert variant="info">
+                    <CircleAlertIcon className="size-5" />
+                    <AlertTitle>
+                      You don't have permission to mint NFTs on this contract
+                    </AlertTitle>
+                  </Alert>
+                )}
+              </AccordionContent>
+            </AccordionItem>
+
+            <AccordionItem
+              value="primary-sale-recipient"
+              className="border-none "
+            >
+              <AccordionTrigger className="border-border border-t px-1 text-left">
+                Claim Conditions & Primary Sale Recipient
+              </AccordionTrigger>
+              <AccordionContent className="px-1">
+                <ConfigSection
+                  isOwnerAccount={props.isOwnerAccount}
+                  primarySaleRecipient={props.primarySaleRecipient}
+                  claimCondition={props.claimCondition}
+                  update={props.update}
+                  isErc721={props.isErc721}
+                  chainId={props.chainId}
+                />
+              </AccordionContent>
+            </AccordionItem>
+          </Accordion>
+        </div>
+      )}
+    </ModuleCardUI>
+  );
+}
+
+const configFormSchema = z.object({
+  primarySaleRecipient: addressSchema,
+
+  tokenId: z.string().optional(),
+
+  pricePerToken: z.number().optional(),
```

Comment:
can not just use `z.number()....` in forms - this will never be satisfied because form values are strings. it should be `z.coerce.number()...`